target/linux/ath79/image/generic.mk

```diff
@@ -171,6 +171,14 @@ define Device/comfast_cf-e120a-v3
 endef
 TARGET_DEVICES += comfast_cf-e120a-v3
 
+define Device/comfast_cf-e314n-v2
+  ATH_SOC := qca9531
+  DEVICE_TITLE := COMFAST CF-E314N v2
```

Comment:
Please update to use DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT as introduced in:
https://github.com/openwrt/openwrt/commit/8c3dbaf8b49233e275b2dce4e30dc0a2bf8c86c5

Reply:
@adrianschmutzler Problem with such setup, is that it does not appear in the "make menuconfig" setup

Reply:
Sounds like you haven't rebased onto recent master?

Reply:
I tested against 19.07
INdeed it works with master, but main branch is 19.07 right ?

Reply:
You should always develop against master.
After 19.07 has been made it's own branch, only bugfixes will be merged there (= stable branch).
Device support patches are (almost) never backported. If you desire to have this device in 19.07, you will have to provide a backport patch in your local repository.